microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -61,8 +61,13 @@ See the following list for details:
 //[id="microshift-4-16-configuring"]
 //===Configuring
 
-//[id="microshift-4-16-networking"]
-//=== Networking
+[id="microshift-4-16-networking"]
+=== Networking
+
+[id="microshift-4-16-multus"]
+==== Multiple networks capability now available
+
+With this release, using multiple networks is supported with the {microshift-short} Multus plugin. If you have advanced networking requirements, you can attach additional networks to a pod for high-performance network configurations. After installing the Multus RPM, you can use the Bridge, MACVLAN, or IPVLAN plugins to create additional networks. See xref:../microshift_networking/microshift_multiple_networks/microshift-cni-multus.adoc#microshift-multus-intro_microshift-cni-multus[Additional networks in {microshift-short}].
```

Comment:
"After installing the Multus RPM..."  Not sure, is there a word missing?  e.g., "the Multus RPM package" ? (This isn't my area so please ignore if not applicable)

Reply:
You're right, that's the style guidance, https://github.com/redhat-documentation/supplementary-style-guide/blob/main/supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc#-rpm-noun 
I think I subconsciously forget that one because saying "Red Hat Package Manager package" seems a wee silly.